test/sentry/client_error_test.exs

```diff
@@ -0,0 +1,55 @@
+defmodule Sentry.ClientErrorTest do
+  use Sentry.Case
+  alias Sentry.ClientError
+
+  describe "message/1" do
+    test "With atom - returns message " do
```

Comment:
Supa dupa nit: let's start tests with a lowercase and no trailing space

```suggestion
    test "with atom - returns message" do
```

in the whole file here.